src/ramses_tx/ramses.py

```diff
@@ -1227,10 +1228,18 @@
     "07": "off",
 }
 
+_22F1_MODE_VASCO: dict[str, str] = {  # for VASCO D60 AND ClimaRad Minibox fanS/remoteS
+    "02": "low",  # low:    000206
+    "03": "medium",  # medium: 000306
+    "04": "high",  # high:   000406, aka boost with 22F3
+    "05": "auto",
+}
+
```

Comment:
What do you have for `00`, `01` and `06`?

If you are brave, you can fake these payloads and see what happens.

For this packet:
```
 I --- 29:091138 32:022222 --:------ 22F1 003 000106
```
... do you see a response from `32:022222` (I have not seen any packets from 32:022222 in the log you sent me).

Reply:
`32:022222` indeed is a rather shy HRU unit. The remotes can bind and command only in the room, and I had to add it as orphan in the config. 
After we have the base of this merged, I will investigate by putting the ramses_esp closer.

Reply:
Added 01 for away/minimum.

Reply:
If I add an `off` entry to the ramses_cc config flow, double quates are added around it. Fine
If I add an entry for 01 called lowlow and submit, my whole known devices is empty. Are some names reserved, or from an enum?

Reply:
> If I add an entry for 01 called lowlow and submit, my whole known devices is empty. Are some names reserved, or from an enum?

No. 

Likely your YAML was formatted incorrectly.